symbulate/results.py

```diff
@@ -309,15 +322,65 @@ def plot(self, type=None, alpha=None, normalize=True, jitter=False, **kwargs):
                 raise Exception("Histogram must have type='impulse' or 'bar'.")
         elif dim == 2:
             x, y = zip(*self)
+
+            x_count = count_var(x)
+            y_count = count_var(y)
+            x_height = x_count.values()
+            y_height = y_count.values()
+            
+            discrete_x = False
```

Comment:
This code can be made more concise:

`discrete_x = sum([(i > 1) for i in x_height]) > .8 * len(x_height)`